openvpn-install.sh

```diff
@@ -681,8 +681,8 @@ function installOpenVPN() {
 	# Install the latest version of easy-rsa from source, if not already
 	# installed.
 	if [[ ! -d /etc/openvpn/easy-rsa/ ]]; then
-		local version="3.0.6"
-		wget -O ~/EasyRSA-unix-v${version}.tgz https://github.com/OpenVPN/easy-rsa/releases/download/v${version}/EasyRSA-unix-v${version}.tgz
+		local version="3.0.7"
+		wget -O ~/EasyRSA-unix-v${version}.tgz https://github.com/OpenVPN/easy-rsa/releases/download/v${version}/EasyRSA-${version}.tgz
 		tar xzf ~/EasyRSA-unix-v${version}.tgz -C ~/
 		mkdir -p /etc/openvpn/easy-rsa
 		mv ~/EasyRSA-v${version}/* /etc/openvpn/easy-rsa/
```

Comment:
@angristan This should be `mv ~/EasyRSA-${version}/* /etc/openvpn/easy-rsa/` without the `v`

Reply:
https://github.com/angristan/openvpn-install/commit/0e961a2e6b362574c705d00b121fbecad4d1de83 :)